src/react-components/src/components/00-base/SectionName.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+
+const SectionName = ({ children }) => <div className={bem('main', ['research'])} >{children}</div>;
```

Comment:
I wonder if "research" is a thing that should be hardcoded, or if this is a thing that should be passed as a modifier from the consumer (so a Research product would pass "research" but a Locations product could pass "locations", etc)

Reply:
That's what I'd like to do but it felt out of scope for this ticket—completely agree though!

Reply:
(To explain _why_ such a small thing wasn't done here, though, is that I'm not entirely sure what all of the site sections are yet. I could create another `enum` just to get this started and clean this up and add on once I've tracked that info down... I will do that, actually)